### ISSUE TITLE
(documentation) DnD with custom Rule and RuleGroup

### ISSUE THREAD

**Issue by unrealsolver** [react-querybuilder/react-querybuilder#449]
By default, rqb/dnd will not work with custom Rule and RuleGroup. To make it work, `controlElements` should be added to `QueryBuilderDnD` instead of `QueryBuilder` component, like this:
```javascript
<QueryBuilderDnD controlElements={controlElements}>
  <QueryBuilder
    query={query}
    // ...
    // Don't put 'controlElements' here!
  />
</QueryBuilderDnD>
```
I think it should be added into documentation, because it is not obvious and could not be figured out without debugging the library internals.
I can update the docs if this point makes sense.

**Comment by jakeboone02** [comment]
Yeah, that's a good point. The docs probably need a dedicated DnD page anyway, but if you'd like to amend the current docs please feel free to submit a PR. Docs are in the `/website/docs` folder, and since this still applies to v5, also update `/website/versioned_docs/version-5`.

Thanks!

**Comment by jakeboone02** [comment]
This will be addressed with a note in the documentation in v6.

**Comment by jakeboone02** [comment]
@unrealsolver I just released [v6.0.0](https://github.com/react-querybuilder/react-querybuilder/releases/tag/v6.0.0) along with an [update to the documentation](https://react-querybuilder.js.org/docs/components/querybuilder#rule) regarding custom `Rule` and `RuleGroup` components when using `QueryBuilderDnD`.
